data_steward/resources.py

```diff
@@ -95,7 +95,7 @@ def is_pii_table(table_id):
     :param table_id: identifies the table
     :return: True if specified table is a pii table, False otherwise
     """
-    return table_id.startswith('pii')
+    return table_id.startswith('pii') or table_id.startswith('participant')
```

Comment:
This is something I am wondering about.  If the participant_match table followed the naming convention of the other pii tables, we wouldn't have to change this line or introduce a new constant string.  `pii_participant_match` doesn't seem inappropriate, as it is describing the match information of participant pii.  Thoughts?

Reply:
I had similar concerns with the name. `pii_validation` or `pii_id_match` would be processed by the existing methods and convey the purpose of the table just as well. If it's easy to change from a business perspective and convey that to the sites, that would be desirable but not necessary. The current modifications should suffice to handle `participant_match`, and I think the word is already out to the sites that we will be using this name. We can always change it later on if required.

Reply:
Fair point.  It seems like `is_pii_table` is only used in one location in the code base.  You could change this to use the defined `PII_TABLE` list like `return table_id in PII_TABLES` and then you wouldn't even need to worry about how the tables are named.  But looking at it further, you wouldn't be able to easily do this without refactoring the constants file because it would create a circular import.  The constants file shouldn't be importing from another module.  This is fine until that can be addressed.

Reply:
DC-358 #comment trying to see if this shows up in jira

Reply:
Yes, the circular import would involve a lot more work to resolve. I went ahead with adding the `participant` prefix since we might have files with that prefix going forward.

Reply:
To me, the fact that we could end up with a circular import from a constants file is an indicator we should clean this up.  Consider it as future work.